server/db/config/build.js

```diff
@@ -5,7 +5,8 @@ const connection = require('./connection');
 
 const dbBuild = () => {
   const sql = readFileSync(join(__dirname, 'build.sql')).toString();
-  return connection.query(sql);
+  return connection.query(sql).then(console.log('Done')).catch((err => console.log(err)));
 };
 
+dbBuild();
```

Comment:
we don' need to call dbBuild function here , we need it for the tests 
so every database test you will call the function in the test file